### ISSUE TITLE
Make parameter identifier character configurable

### ISSUE THREAD

**Issue by mrcolumbia** [dbeaver/dbeaver#35052]
Right now, parameters are identified like so: ${parameter}. Is it possible to make the $ character something like & instead?

**Comment by E1izabeth** [comment]
We have also parameters identified with `:`. They works the same way and you can change the prefix in preferences
![image](https://github.com/user-attachments/assets/daf9fec4-eeda-400d-9cb3-02f7ee8ab013)

**Comment by mrcolumbia** [comment]
Yes, I’m aware of this but I want it with the bracket structure.

**Comment by E1izabeth** [comment]
Then it's not possible now.

**Comment by E1izabeth** [comment]
Thank you for request
